docs/openapi.yaml

```diff
@@ -1693,13 +1693,31 @@ paths:
             `id` of the instance to be fetched
           schema:
             type: string
+        - name: limit
```

Comment:
Docs is not properly updated, you have to mention in introduction or in some other header that this bla bla api pagination introduced with maximum value and by default value to be allowed. Check other server docs